src/main/java/com/pet/bankservice/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package com.pet.bankservice.service.impl;
+
+import com.pet.bankservice.entity.User;
+import com.pet.bankservice.exception.DataProcessingException;
+import com.pet.bankservice.repository.UserRepository;
+import com.pet.bankservice.service.UserService;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class UserServiceImpl implements UserService {
+    private final UserRepository userRepository;
+
+    @Override
+    public User save(User user) {
```

Comment:
as far as I know
`update()` uses `save()` method from crud interface but I see that your intention is reusing `save()` method from service. Either you just forgot it or doing wrong - you need `update()` method in service

Reply:
Task: 7. You can have a single save() method on the service layer for both ‘create’ and ‘update’ operations.